script.js

```diff
@@ -1,4 +1,447 @@
+// ____________________  Global Variables ________________________
+
+// L1 Game Mode variables to choose which game user wants to play
+var L1_PICK_GAME_MODE = "Choose your game";
+var L1_NORMAL_MODE = "Normal Mode";
+var L1_LOWEST_MODE = "Lowest Win";
+var L1_AUTO_MODE = "Auto";
+var L1_VARIABLE_DICE_MODE = "Variable Dice";
+var L1_VARIABLE_PLAYERS_MODE = "Variable Players";
+
+// modes to faciliate running of games
+var L2_ROLL_DICE = "Roll Dice Mode";
+var L2_ROLL_DICE_AUTO = "Roll and Auto Pick";
+var L2_CHOOSE_ORDER = "Choose Order Mode";
+var L2_DECIDE_WINNER = "Decide Winner";
+var L2_CHOOSE_NUM_DICE = "Choose number of dice";
```

Comment:
Great use of your scream case constants Eric!
And the variable names are clear as well 👍